npc/re/merchants/shops.txt

```diff
@@ -95,7 +95,7 @@ mora,138,110,4	shop	Mora Chef	518,11519:-1
 // Morroc
 //=======================================================
 morocc,156,76,4	shop	Butcher#moc1	58,517:-1
-moc_ruins,91,128,4	shop	Tool Dealer#moc1	93,1750:-1,611:-1,501:-1,502:-1,503:-1,504:-1,506:-1,645:-1,656:-1,601:-1,602:-1,1065:-1
+moc_ruins,91,128,4	shop	Tool Dealer#moc1	93,1750:-1,611:-1,501:-1,502:-1,503:-1,504:-1,645:-1,656:-1,713:-1,601:-1,602:-1,1065:-1
```

Comment:
On iRO it has both green potions and empty bottles, on pre-re are the same items but without bottles.
On aegis 15.2 this npc is completely absent :/